custom_hostname.go

```diff
@@ -55,8 +55,19 @@ type CustomHostnameListResponse struct {
 // hostname in the given zone.
 //
 // API reference: https://api.cloudflare.com/#custom-hostname-for-a-zone-update-custom-hostname-configuration
-func (api *API) UpdateCustomHostnameSSL(zoneID string, customHostnameID string, ssl CustomHostnameSSL) (CustomHostname, error) {
-	return CustomHostname{}, errors.New("Not implemented")
+func (api *API) UpdateCustomHostnameSSL(zoneID string, customHostnameID string, ssl CustomHostnameSSL) (*CustomHostnameResponse, error) {
```

Comment:
Is there a reason you're changing return type?

Reply:
I just wanted to follow same style as it is in createCustomHostname, reponse from the API is similar on borh cases.